### ISSUE TITLE
[BUG] Unable to control RGBW

### ISSUE THREAD

**Issue by bigboban** [StyraHem/ShellyForHASS#634]
## Environment

- ShellyForHASS release with the issue: 0.3.3
- Last working ShellyForHASS release (if known): Not sure, but few weeks ago.
- Home Assistant Core release with the issue: 2021.12.0
- Operating environment (Home Assistant/Supervised/Docker/venv): hassio supervised

## Describe the bug

Unable to control any command to RGBW device. Unable to change color nor switch on/off.

## Steps to Reproduce

If device is off and i click ON (slider) nothing happens and slider changes back to off. But other direction works fine. If i switch on/off or change color using shelly android app or RGBW web interface, right state is changed in HA. And change is very fast so i think coap is working.

## Expected behavior

Can control RGBW from HA.

## Traceback/Error logs

I found no ERROR in log about this.

```txt

```

## Additional context

RGBW fw version 20211109-130054/v1.11.7-g682a0db and it states it is last.

Other shellies works fine from HA and control is without any problem. Only RGBW has this problem.

**Comment by hummer2k** [comment]
Same problem with dimmer1 and dimmer2. 

Last working ShellyForHass version: **0.2.2**

Tested shelly firmware versions:

**Dimmer1**:
    
- `20211109-130937/v1.11.7-g682a0db`
    
**Dimmer2**: 

- `20211109-131033/v1.11.7-g682a0db`
- `20210429-102101/v1.10.4-g3f94cd7`
- `20210115-103951/v1.9.4@e2732e05`

**Comment by bigboban** [comment]
Too late for me. I cannot wait so i switched to official Shelly integration in HA. It has some advantages and some disadvantages but - RGBW works fine!

**Comment by devconprodcutions** [comment]
I have the same issue since today. All Shellies except my rgbw2 shellies (two) cannot be Controlled anymore. They are Shown as Not available. Shelly4Hass(0.32 and 0.33 tested). HA 2021.12.7.

**Comment by seb-koch** [comment]
Yes, same here my RGBW canÄt be controlled anymore. The state of the LED's are transmitted correctly from Shelly RGBW2 to the Home Assistant server, when I dim from the shelly web interface. Any changes made in HASS aren't transmitted to the shelly, no matter if from lovelace or from within a scene.

**Comment by seb-koch** [comment]
I am happy to help debugging it. Let me know what I may do to provide the right logs or details.

**Comment by lexathon** [comment]
I seem to have this too. Lots of shelly devices on my system and a whole bunch of dimmers, rgbs no longer controllable. I see their status but can't control. Was working fine though until a reboot. Nothing in the log.

**Comment by lexathon** [comment]
Rolled back to 0.3.1 and all seems to be well again.
